testsuite/features/step_definitions/api_common.rb

```diff
@@ -570,3 +575,11 @@ def wait_action_complete(actionid, timeout: DEFAULT_TIMEOUT)
 Then(/^"([^"]*)" should be present in the result$/) do |profile_name|
   assert($output.select { |p| p['name'] == profile_name }.count == 1)
 end
+
+When(/^I create and modify the kickstart system "([^"]*)" with hostname "([^"]*)" via XML-RPC$/) do |name, hostname, values|
+  system_id = $api_test.system.create_system_profile(name, 'hostname' => hostname)
+  STDOUT.puts "system_id: #{system_id}"
+  # this works only with a 2 column table where the key is in the left column
+  variables = values.rows_hash
+  _command = $api_test.system.set_variables(system_id, variables)
```

Comment:
Very nitpicky, feel free to ignore - the _command variable name seems like a weird choice to me at first glance - is the output of that function a command? in any case, since we're not using the output, I think it'd just be easier to name the variable `_output` to avoid confusion

Reply:
Thank you for the suggestion. I completely removed the assignment to a variable since we do not need and use it.